roles/provision-v2v/tasks/login.yml

```diff
@@ -4,3 +4,6 @@
 
 - name: Change project to {{ namespace }}
   command: "oc project {{ namespace }}"
+
+- name: Add priviliged policy for User
+  command: "oc adm policy add-scc-to-user privileged system:serviceaccount:{{ namespace }}:kubevirt-privileged"
```

Comment:
What if this exists already?  Seems a common thing I've seen in playbooks?  Isn't it kubevirt's job to give its users appropriate permissions?

Reply:
Ok.  so just run this pod as the kubevirt-privileged serviceaccount.